app/uk/gov/hmrc/taxenrolmentassignmentfrontend/reporting/AuditEvent.scala

```diff
@@ -59,36 +51,89 @@ object AuditEvent {
       auditType = "ReportUnrecognisedAccount",
       transactionName = "reporting-unrecognised-pt-account",
       detail = getDetailsForReportingFraud(
-        request.userDetails,
-        suspiciousAccountDetails,
-        AccountTypes.PT_ASSIGNED_TO_OTHER_USER
+        suspiciousAccountDetails
       )
     )
   }
 
+  def auditSuccessfullyAutoEnrolledPersonalTax(accountType: AccountTypes.Value)
+                                              (implicit request: RequestWithUserDetailsFromSession[_]): AuditEvent = {
+
+    val optSACredentialId = if(request.userDetails.hasSAEnrolment || accountType == SA_ASSIGNED_TO_CURRENT_USER) {
+      Some(request.userDetails.credId)
```

Comment:
This audit event is when the auto enrolment has happened rather than the user making choosing to enrol with the current signed in account rather than the account which has SA. Auto enrolment only happens for users with multiple accounts if they have no SA enrolment or if they are currently signed in with the account with SA. L76 function is specific to users that have SA on another account which is why the session data is used to obtain the details.